include/numerics/numeric_vector.h

```diff
@@ -723,7 +723,8 @@ class NumericVector : public ReferenceCountedObject<NumericVector<T>>,
    * PetscVectors.
    */
   virtual void create_subvector(NumericVector<T> & ,
-                                const std::vector<numeric_index_type> &) const
+                                const std::vector<numeric_index_type> &,
+                                bool = true) const
```

Comment:
My very poor excuse for why I didn't do this initially is that I will have to comment out the parameter name to avoid unused warnings. Will a commented out parameter name still show up in the doxygen? Either way I can still articulate something in the doxygen wrt that parameter

Reply:
relevant issue: https://github.com/doxygen/doxygen/issues/6926